src/ralph/back_office/migrations/0005_warehouse_stocktaking_tag_suffix.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('back_office', '0004_warehouse_stocktaking_enabled'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='warehouse',
+            name='stocktaking_tag_suffix',
+            field=models.CharField(default='WHOUSE', max_length=8),
```

Comment:
regenerate migrations (remove this and create new one) - this one still has default value `WHOUSE`